tests/really-object-spec.coffee

```diff
@@ -0,0 +1,231 @@
+# Dependencies
+
+Q            = require 'q'
+ReallyObject = require '../src/really-object'
+ReallyError  = require '../src/really-error'
+protocol     = require '../src/protocol'
+Websocket    = require '../src/transports/webSocket'
+CONFIG       = require './support/server/config'
+Really       = require '../src/really'
+
+describe 'really-object', ->
+
```

Comment:
You just need to mock `websocket` as we don't need to initialize real `websocket` in the tests...

try this
channel =
  send: -> true